### PR TITLE
Fix: avoid NaN if no attendances have been registered

### PR DESCRIPTION
**Pull request by EWLameijer**

### CODE REVIEW

**Review by MtePoele — CHANGES_REQUESTED** [review]
This looks awesome and works great!

I tested this by deleting the DB and having it get seeded. This had the minor inconvenience that today did get attendances. So I figured "I'll just pick a future date to see of NaN is gone".
And it was gone. Awesome.

However, a "from date" can't be in the future. And so I realized I forgot to prevent that when I made this feature.
Turns out there's a "max" that can be set to today.
https://stackoverflow.com/questions/32378590/set-date-input-fields-max-date-to-today

Might we worth adding this?

File: backend/src/main/java/nl/itvitae/attendancetracker/attendance/AttendanceController.java [hunk 1]
```diff
@@ -48,11 +48,14 @@ public class AttendanceController {
...
         var attendanceRegistrations = attendanceRegistrationRepository.findByAttendanceStudentId(studentId);
         var attendances = attendanceRegistrations.stream().map(AttendanceRegistration::getAttendance).distinct();
+        var today = LocalDate.now();
         return attendances.map(
                         attendance -> attendanceRegistrations.stream()
                                 .filter(attendanceRegistration -> attendanceRegistration.getAttendance() == attendance)
                                 .max(Comparator.comparing(AttendanceRegistration::getDateTime)))
-                .flatMap(Optional::stream).map(AttendanceRegistrationDto::from).toList();
+                .flatMap(Optional::stream)
+                .filter(attendanceRegistration -> !attendanceRegistration.getAttendance().getDate().isAfter(today))
```
Comment (MtePoele) [inline backend/src/main/java/nl/itvitae/attendancetracker/attendance/AttendanceController.java:57]: ! ... isAfter(today)
Leest dat niet fijner als 
... isBefore(today)?

**Review by MtePoele — APPROVED** [review]
Might be worth getting rid of the double negative (see comment).

Other than that it looks and works great.

File: backend/src/main/java/nl/itvitae/attendancetracker/attendance/AttendanceController.java (hunk 1 above)
Comment (EWLameijer) [reply backend/src/main/java/nl/itvitae/attendancetracker/attendance/AttendanceController.java:57]: not after  !== before (as I want to include today in the overview)

**Pull request merged**